Add Swagger in User guide for Catalog Registration Chapter (#3738)
SkylineCommunications/dataminer-docs: user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md

The register API call allows you to create or update a Catalog item. To add a version after you have successfully registered an item, see [Registering a new version with the API](#registering-a-new-version-with-the-api).

### API Definition

For a complete definition of the API, go to [Key Catalog API Swagger](https://catalogapi-prod.cca-prod.aks.westeurope.dataminer.services/swagger/index.html?urls.primaryName=Key+Catalog+API+v1.0).

This page also provides an easy way to try out the call: Expand the "catalog/register" item, and click the *Try it out* button.

> [!IMPORTANT]

> > The *Try it out* button allows you to execute the register call on the production Catalog.

Author: indeed. you will perform an action on the actual Catalog data

> > The *Try it out* button allows you to execute the register call on the production Catalog.

Reviewer: Doesn't this make the "Try it out" title of the button a little misleading? Users might get the impression that they're just doing a harmless test, but they're actually making changes in the Catalog.

> > The *Try it out* button allows you to execute the register call on the production Catalog.

Author: I can see people can have that impression but we don't have full control on what is being shown on that page so unfortunately we cant change the text. Best alternative is perhaps improving that warning message? I do believe it will be clear for people familiar with swagger/apis

> This page also provides an easy way to try out the call: Expand the "catalog/register" item, and click the *Try it out* button.
> 
> > [!IMPORTANT]
> > The *Try it out* button allows you to execute the register call on the production Catalog.

Reviewer: How about this?

```suggestion
This page also provides a quick way to execute the call: Expand the "catalog/register" item, and click the *Try it out* button.

> [!IMPORTANT]
> Clicking the *Try it out* button will execute the register call on the production Catalog.
```

Author: looks clear , its good
